midi.zig

```diff
@@ -2,25 +2,27 @@ const std = @import("std");
 
 const mem = std.mem;
 
+const midi = @This();
+
 pub const decode = @import("midi/decode.zig");
 pub const encode = @import("midi/encode.zig");
 pub const file = @import("midi/file.zig");
 
+pub const File = file.File;
+
 test "midi" {
     _ = @import("midi/test.zig");
     _ = decode;
     _ = file;
 }
 
-pub const File = file.File;
-
 pub const Message = struct {
     status: u7,
     values: [2]u7,
 
     pub fn kind(message: Message) Kind {
-        const _kind = @truncate(u3, message.status >> 4);
-        const _channel = @truncate(u4, message.status);
+        const _kind = @as(u3, @truncate(message.status >> 4));
+        const _channel = @as(u4, @truncate(message.status));
```

Comment:
These are all over. I recommend specifying the type of the variable instead of using `@as`

```suggestion
        const _kind: u3 = @truncate(message.status >> 4);
        const _channel: u4 = @truncate(message.status);
```